src/fsharp/CompileOps.fsi

```diff
@@ -160,8 +161,9 @@ type IRawFSharpAssemblyData =
     abstract ShortAssemblyName: string
 
 type TimeStampCache = 
-    new: defaultTimeStamp: DateTime -> TimeStampCache
+    new: defaultTimeStamp: DateTime * [<Optional; DefaultParameterValue(false)>] isInteractive: bool -> TimeStampCache
```

Comment:
Just make this a non-optional argument, we may as well make it explicit everywhere